src/nginxconfig/templates/setup_sections/ssl_cert.vue

```diff
@@ -82,15 +92,29 @@ THE SOFTWARE.
 
             <li>
                 <p>
-                    {{ $t('templates.setupSections.certbot.configureCertbotToReloadNginxOnCertificateRenewal') }}
+                    {{ $t('templates.setupSections.ssl_cert.configureSslCertToReloadNginxOnCertificateRenewal') }}
                     <br />
                 </p>
+                <p>
+                    Certbot:
+                </p>
                 <BashPrism cmd="echo -e '#!/bin/bash\nnginx -t && systemctl reload nginx' | sudo tee /etc/letsencrypt/renewal-hooks/post/nginx-reload.sh"
                            @copied="codeCopiedEvent('Create nginx auto-restart on renewal')"
                 ></BashPrism>
                 <BashPrism cmd="sudo chmod a+x /etc/letsencrypt/renewal-hooks/post/nginx-reload.sh"
                            @copied="codeCopiedEvent('Enable execution of auto-restart')"
                 ></BashPrism>
+                <p>
+                    Acme.sh:
+                </p>
+                <BashPrism :key="acmeCopy"
+                           :cmd="acmeCopy"
+                           @copied="codeCopiedEvent('Move SSL certificates to Let`s Encrypt directory')"
+                ></BashPrism>
```

Comment:
This step needs to be part of the obtaining certificates step, so that the certs are in the right place for when the nginx config is reloaded, otherwise it'll fail?